bin/clean-tsv-metadata

```diff
@@ -36,10 +41,15 @@ if __name__ == "__main__":
             type=int,
             help="Number of columns to enforce.\n")
 
+    parser.add_argument(
+            "--sort-col",
+            type=str,
+            help="A column name (or 1-based index for headerless tsvs) to prioritize sorting; if not used, sorting will be alphabetical by columns in the order they appear.`.\n")
```

Comment:
Why did you choose to use 1-based indexing? I would assume that most python users are familiar with 0-based indexing and may even overlook this point.

Reply:
You're right, changed it in 74c4dcb!